doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -656,26 +671,56 @@ In a Coq term, writing :g:`$x` is semantically equivalent to
 :g:`ltac2:(Control.refine (fun () => x))`, up to re-typechecking. It allows to
 insert in a concise way an Ltac2 variable of type :n:`constr` into a Coq term.
 
+Match
+~~~~~
+
+.. tacn:: match @tac2expr5 with {? @ltac2_branches } end
+   :name: match (Ltac2)
+
+   .. insertprodn ltac2_match atomic_tac2pat
+
+   .. prodn::
+      ltac2_match ::= match @tac2expr5 with {? @ltac2_branches } end
```

Comment:
If this `match` is mostly an implementation-detail, then it should probably not be presented first, without explanations. Also, the repetition between the `tacn` and the first line of the `prodn` is puzzling. The former should probably be removed.

Reply:
I think it's roughly the same as an OCaml match.  I'll move it down.

> Also, the repetition between the `tacn` and the first line of the `prodn` is puzzling. The former should probably be removed.

That makes it unfindable in the tactic index.